### ISSUE TITLE
--help shows undefined

### ISSUE THREAD

**Issue by AndreasNasman** [ty-ras/meta#6]
When running `npx @ty-ras/start --help`, an unexpected `undefined` is shown.

```sh
❯ npx @ty-ras/start --help

  Usage: npx undefined [options...] [folder] # 👈 Unexpected `undefined`.

  All options and folder are optional as command-line arguments.
  If any of them is omitted, the program will prompt for their values.
  Options:
    --components, -p    Which components will be using TyRAS libraries?
          Schema: "be"|"fe"|"be-and-fe"
    --dataValidation, -d        Which data validation framework should TyRAS be providing?
          Schema: "io-ts"|"zod"
    --server, -s        Which server should TyRAS be providing?
          Used only when components is "be" or "be-and-fe".
          Schema: "node"
    --client, -c        Which client should TyRAS be providing?
          Used only when components is "fe" or "be-and-fe".
          Schema: "fetch"
```

**Comment by AndreasNasman** [comment]
Looks like the [(await readPkgUp.readPackageUp())?.packageJson.name](https://github.com/ty-ras/meta/blob/main/start/src/collect-input/cli-args.mts#L62) here returns the `undefined` 🤔

**Comment by stazz** [comment]
From [docs](https://www.npmjs.com/package/read-pkg-up):

```
cwd
Type: URL | string
Default: process.cwd()

The directory to start looking for a package.json file.
```

Makes sense... At least fix is thus somewhat easy. 👍

**Comment by stazz** [comment]
Fixed in version `0.13.1` 👍
